FrontEnd/src/PostFullDisplay.jsx

```diff
@@ -1,14 +1,31 @@
 import { useNavigate } from "react-router-dom";
+import {useState} from "react";
 import "./PostFullDisplay.css";
-function PostFullDisplay({ userUID, post, isShown, onClose }) {
+
+function PostFullDisplay({ userUID, post, isShown, onClose, userData, socket }) {
+  const [favorited, setFavorited] = useState(false)
   const navigate = useNavigate();
   const handleBookingFormOpen = () => {
     navigate("/BookingPage", { state: { post, userUID } });
   };
-  if (!isShown) {
+  if (isShown === false) {
```

Comment:
Why did you change that? Is `isShown` nullable? If not, the previous version is better.